src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -29,44 +29,50 @@ public sealed class MatrixFactorizationTrainer : TrainerBase<MatrixFactorization
     {
         public sealed class Arguments
         {
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Regularization parameter")]
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Regularization parameter." +
+                "It's the weight of factor matrices' norms in the objective function minimized by matrix factorization's algorithm." +
+                "A amall value could cause over-fitting.")]
             [TGUI(SuggestedSweeps = "0.01,0.05,0.1,0.5,1")]
             [TlcModule.SweepableDiscreteParam("Lambda", new object[] { 0.01f, 0.05f, 0.1f, 0.5f, 1f })]
-            public Double Lambda = 0.1;
+            public double Lambda = 0.1;
 
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Latent space dimension")]
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Latent space dimension (denoted by k). If the factorized matrix is m-by-n," +
+                "two factor matrices found by matrix factorization are m-by-k and k-by-n, respectively." +
```

Comment:
Same note. In fact review all of the places where you've split your documentation strings across multiple lines, since they are incorrect more often than they are correct.